Add initial 2024 Survey Landing (#59)
gradle/community: docs/surveys/developer-survey/README.md

---
title: "Gradle Build Tool - 2024 Software Developer Survey"
description: >

>   We're doing a developer survey to prioritize our developer experience work.

Reviewer: ```suggestion
  We've created a survey to help us better understand developers' experiences using Gradle Build Tool.
```

>   Take the survey and get gift cards and special swag!

Reviewer: ```suggestion
  To say thank you to developers who complete the survey, we'll be handing out gift cards and special swag!
```

---

# Gradle Build Tool - Software Developer Survey

The Gradle team is committed to enhancing developer experience,

> it is one of the top priorities on [Gradle Roadmap](../..roadmap/README.md/).

Reviewer: ```suggestion
as one of the top priorities on the [Gradle Roadmap](../..roadmap/README.md/).
```

Your feedback is crucial in helping us prioritize our efforts!

<a class="button button--blue"
   href="https://www.surveymonkey.com/r/7FZPXQZ"
   target="_blank">Take the Survey
</a>

> ## 2024 Developer Survey

Reviewer: ```suggestion
## The Survey
```

![Test](./images/hero.png)

> The 2024 survey focuses on Gradle Build Tool developer experience, intends to get feedback on complexity of particular operations and to prioritize them.

Reviewer: ```suggestion
The 2024 survey focuses on Gradle Build Tool developer experience.  We're aiming to discover common pain points and understand what use cases are particularly frustrating.
```

Author: Will add it but will restore my sentence about prioritization, too

> This is why the form is quite long, and we will appreciate your patience.

Reviewer: ❗ I don't think we should say this at all.  Most of our sample respondents completed the original version in 10-15 min, and we think the version you revised should be even faster than that on average.

Author: Everything beyond 2 minutes is long. Happy to remove/rework though

Reviewer: Regardles of if it _is_ long or not (which I guess depends on how you define long), we shouldn't _advertise_ it as being long, as this might discourage people from responding.

> The 2024 survey focuses on Gradle Build Tool developer experience.  We're aiming to discover common pain points and understand what use cases are particularly frustrating.
> This is why the form is quite long, and we will appreciate your patience.

Author: ```suggestion
```

The survey will be open for responses until the end of the year,
and we intend to make it a recurrent event.

<a class="button button--blue"
   href="https://www.surveymonkey.com/r/7FZPXQZ"
   target="_blank">Take the Survey
</a>

## Swag

We offer a gift card raffle for anonymous responses.

> Several most active participants, who provide substantial feedback and have a follow-up with us,

Reviewer: ```suggestion
The most active participants, who provide substantial feedback and participate in follow-up interviews with us,
```

will get special edition Gradle swag.
To participate in the raffle,
submit your feedback by October 31st, 2024.

## Previous Surveys

> It is our first one!

Reviewer: 🤔 This seems odd, I'd consider dropping it.

Author: I was about verify the survey history and link them if I find something
